lib/modules/platform/github/index.ts

```diff
@@ -76,19 +76,26 @@ import type {
   PlatformConfig,
   PrList,
 } from './types';
-import { UserDetails, getUserDetails, getUserEmail } from './user';
+import { getUserDetails, getUserEmail } from './user';
 
 const githubApi = new githubHttp.GithubHttp();
 
-let config: LocalRepoConfig = {} as any;
+let config: LocalRepoConfig;
+let platformConfig: PlatformConfig;
 
-const platformConfig: PlatformConfig = {
-  hostType: PlatformId.Github,
-  endpoint: 'https://api.github.com/',
-};
+export function resetConfigs(): void {
```

Comment:
I don't like to create it only for tests, but seems like eventually we'll have class-based platform code and this one will go away